### ISSUE TITLE
Error in structured data

### ISSUE THREAD

**Issue by ankitpokhrel** [easydigitaldownloads/easy-digital-downloads#3572]
Looks like there is a problem with the structured data edd generates. Google says _The property price is not recognised by Google for an object of type SoftwareApplication._

![EDD Structured Data](http://ankitpokhrel.com.np/img/edd-structured-data.png)

The price should go inside itemprop offers
```
<div itemprop="offers" itemscope itemtype="http://schema.org/Offer">
    <span itemprop="price">$25.00</span>
    <meta itemprop="priceCurrency" content="USD" />
    <link itemprop="availability" href="http://schema.org/InStock" />In stock
</div>
```
Also a recommended field _operatingSystem_ is missing.

**Comment by pippinsplugins** [comment]
Thanks for the headsup!

Related: #3355 and #3537

**Comment by pippinsplugins** [comment]
@ankitpokhrel what version of EDD are you using? We already have it setup like that: https://github.com/easydigitaldownloads/Easy-Digital-Downloads/blob/master/includes/template-functions.php#L254

**Comment by ankitpokhrel** [comment]
@pippinsplugins I am using v2.4.2. Still getting the same error in [structured data test tool](https://developers.google.com/structured-data/testing-tool/) for [this download](http://ankitpokhrel.com.np/blog/downloads/dynamic-featured-image-pro/) which has variable pricing enabled. `edd_add_schema_microdata()` seems to be false so the itemprop attribute is missing. Or is there anything in settings to enable schema microdata :smile:?

**Comment by pippinsplugins** [comment]
@ankitpokhrel on a variable priced download or single price?

**Comment by ankitpokhrel** [comment]
@pippinsplugins only on a variable priced download. No problem with the single priced download.

**Comment by pippinsplugins** [comment]
The `name` item appears to be missing too: https://wordpress.org/support/topic/add-schema-name-to-download-items

**Comment by pippinsplugins** [comment]
Another related PR: https://github.com/easydigitaldownloads/easy-digital-downloads/pull/4567

**Comment by pippinsplugins** [comment]
Merged #4567 into `release/2.6`
